draft-js-hashtag-plugin/src/utils/index.js

```diff
@@ -0,0 +1,36 @@
+import regexes from './hashtagRegex';
+
+/* eslint-disable */
+export function extractHashtagsWithIndices(text) {
+  if (!text || !text.match(regexes.hashSigns)) {
+    return [];
+  }
+
+  var tags = [];
+
+  text.replace(regexes.validHashtag, function (match, before, hash, hashText, offset, chunk) {
+    const after = chunk.slice(offset + match.length);
+    if (after.match(regexes.endHashtagMatch)) {
+      return;
+    }
+    const startPosition = offset + before.length;
+    const endPosition = startPosition + hashText.length + 1;
+    tags.push({
+      hashtag: hashText,
+      indices: [startPosition, endPosition],
+    });
+  });
+
+  return tags;
+}
+
+export function extractHashtags(text) {
```

Comment:
I used this for testing initially. It's not necessarily for the plugin, I'll take it out.